app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,7 @@
   </head>
   <body>
     <div class="container-fluid">
+      <%= render 'layouts/header' %>
```

Comment:
Hello @theB264 , I tried to rebase my PR, but I don't see the application_spec.rb file. I follow the Nik's guide of git but it didn't work. Can you help me with this?